quantinuum_schemas/models/quantinuum_systems_noise.py

```diff
@@ -14,33 +14,76 @@ class UserErrorParams(BaseModel):
 
     # Physical Noise
     p1: Optional[float] = None
+    """Probability of a fault occurring during a 1-qubit gate."""
```

Comment:
This kind of docstring seems to play nicely with editors, thought it would be helpful to include them here.